@ecomplus/storefront-components/src/js/ProductCard.js

```diff
@@ -113,6 +119,16 @@ export default {
       return checkOnPromotion(body)
         ? Math.round(((body.base_price - getPrice(body)) * 100) / body.base_price)
         : 0
+    },
+
+    isNew () {
+      const { body } = this
+      const today = new Date()
+      const createdAt = new Date(body.created_at)
+      const differenceDates = today.getTime() - createdAt.getTime()
```

Comment:
Não precisa criar uma instância Date `today` pra pegar o timestamp corrente, `Date.now()` tá aí pra isso 😄 

```suggestion
      const differenceDates = Date.now() - new Date(body.created_at).getTime()
```